Microsoft.Azure.Cosmos/src/Linq/CosmosLinqSerializerType.cs

```diff
@@ -0,0 +1,47 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+namespace Microsoft.Azure.Cosmos
+{
+    /// <summary>
+    /// Serializer type to be used for LINQ query translations.
+    /// </summary>
+    /// <example>
+    /// This example creates a <see cref="CosmosLinqSerializerOptions"/>, which specifies a Linq serialization
+    /// strategy.  This is passed to <see cref="Container.GetItemLinqQueryable"/> to apply the specified
+    /// serializer.
+    /// <code language="c#">
+    /// <![CDATA[
+    /// CosmosLinqSerializerOptions options = new()
+    /// {
+    ///        LinqSerializerType = CosmosLinqSerializerType.Custom
+    /// };
+    /// 
+    /// Book book = container.GetItemLinqQueryable<Book>(true, linqSerializerOptions: options)
+    ///                      .Where(b => b.Title == "War and Peace")
+    ///                     .AsEnumerable()
+    ///                     .FirstOrDefault();
+    /// ]]>
+    /// </code>
+    /// </example>
+#if PREVIEW
+    public
+#else
+    internal
+#endif
+        enum CosmosLinqSerializerType
+    {
+        /// <summary>
+        /// This honors Newtonsoft attributes, followed by DataContract attributes. This will ignore System.Text.Json attributes.
```

Comment:
The text assumes the custom serializer is System.Text.Json, but it could be anything, even a custom Newtonsoft serializer?

```suggestion
        /// This honors default serializer (Newtonsoft.Json) attributes, followed by DataContract attributes. This will ignore custom serializer related attributes.
```

Reply:
It could be, yes.  In any case, the default serializer ignores System.Text.Json, which seemed like a point of confusion for users.

Reply:
Updated comment per your suggestion